cmd/exec.go

```diff
@@ -85,9 +101,24 @@ func execRun(cmd *cobra.Command, args []string) error {
 }
 
 func getAWSEnvVars(assumeRoleOutput *sts.AssumeRoleWithWebIdentityOutput) []string {
-	return []string{
+
+	awsEnv, err := loadAWSDefaultEnv()
+	if err != nil {
+		logrus.Debug(err)
+	}
+
+	envVars := []string{
 		fmt.Sprintf("AWS_ACCESS_KEY_ID=%s", string(*assumeRoleOutput.Credentials.AccessKeyId)),
 		fmt.Sprintf("AWS_SECRET_ACCESS_KEY=%s", string(*assumeRoleOutput.Credentials.SecretAccessKey)),
 		fmt.Sprintf("AWS_SESSION_TOKEN=%s", string(*assumeRoleOutput.Credentials.SessionToken)),
 	}
+
+	if awsEnv.DEFAULT_OUTPUT != "" {
+		envVars = append(envVars, fmt.Sprintf("AWS_DEFAULT_OUTPUT=%s", awsEnv.DEFAULT_OUTPUT))
+	}
+	if awsEnv.DEFAULT_REGION != "" {
+		envVars = append(envVars, fmt.Sprintf("AWS_DEFAULT_REGION=%s", awsEnv.DEFAULT_REGION))
+	}
+
+	return envVars
```

Comment:
I think this might not be doing what you intend it to. If you see [here](https://github.com/chanzuckerberg/aws-oidc/pull/55/files#diff-aed93a53379775027b6616ae133e3815R97) we are already appending the existing environment to the commands.

 I think the bug is that we should try to read the `region` and `output` keys from the ~/.aws/config for the `env` and `exec` commands and then, if they are present, export them to the environment.